addons/l10n_mn/data/account.account.tag.csv

```diff
@@ -1,94 +1,164 @@
+account.account_tag_investing,"CF:[1.1.3] Даатгалын нөхвөрөөс хүлээн авсан мөнгө",accounts,
+account_cashflow_tag_114,"CF:[1.1.4] Буцаан авсан албан татвар",accounts,
+account_cashflow_tag_115,"CF:[1.1.5] Татаас, санхүүжилтийн орлого",accounts,
+account_cashflow_tag_116,"CF:[1.1.6] Бусад мөнгөн орлого",accounts,
+account_cashflow_tag_121,"CF:[1.2.1] Ажиллагчдад төлсөн",accounts,
+account_cashflow_tag_122,"CF:[1.2.2] Нийгмийн даатгалын байгууллагад төлсөн",accounts,
+account_cashflow_tag_123,"CF:[1.2.3] Бараа материал худалдан авахад төлсөн",accounts,
+account_cashflow_tag_124,"CF:[1.2.4] Ашиглалтын зардалд төлсөн",accounts,
+account_cashflow_tag_125,"CF:[1.2.5] Түлш шатахуун, тээврийн хөлс, сэлбэг хэрэгсэлд төлсөн",accounts,
+account_cashflow_tag_126,"CF:[1.2.6] Хүүний төлбөрт төлсөн",accounts,
+account_cashflow_tag_127,"CF:[1.2.7] Татварын байгууллагад төлсөн",accounts,
+account_cashflow_tag_128,"CF:[1.2.8] Даатгалын төлбөрт төлсөн",accounts,
+account_cashflow_tag_129,"CF:[1.2.9] Бусад мөнгөн зарлага",accounts,
+account_cashflow_tag_211,"CF:[2.1.1] Үндсэн хөрөнгө борлуулсны орлого",accounts,
+account_cashflow_tag_212,"CF:[2.1.2] Биет бус хөрөнгө борлуулсны орлого",accounts,
+account_cashflow_tag_213,"CF:[2.1.3] Хөрөнгө оруулалт борлуулсны орлого",accounts,
+account_cashflow_tag_214,"CF:[2.1.4] Бусад урт хугацаат хөрөнгө борлуулсны орлого",accounts,
+account_cashflow_tag_215,"CF:[2.1.5] Бусдад олгосон зээл, мөнгөн урьдчилгааны буцаан төлөлт",accounts,
+account_cashflow_tag_216,"CF:[2.1.6] Хүлээн авсан хүүний орлого",accounts,
+account_cashflow_tag_217,"CF:[2.1.7] Хүлээн авсан ногдол ашиг",accounts,
+account_cashflow_tag_221,"CF:[2.2.1] Үндсэн хөрөнгө олж эзэмшихэд төлсөн",accounts,
+account_cashflow_tag_222,"CF:[2.2.2] Биет бус хөрөнгө олж эзэмшихэд төлсөн",accounts,
+account_cashflow_tag_223,"CF:[2.2.3] Хөрөнгө оруулалт олж эзэмшихэд төлсөн",accounts,
+account_cashflow_tag_224,"CF:[2.2.4] Бусад урт хугацаат хөрөнгө олж эзэмшихэд төлсөн",accounts,
+account_cashflow_tag_225,"CF:[2.2.5] Бусдад олгосон зээл болон урьдчилгаа",accounts,
+account_cashflow_tag_311,"CF:[3.1.1] Зээл авсан, өрийн үнэт цаас гаргаснаас хүлээн авсан",accounts,
+account_cashflow_tag_312,"CF:[3.1.2] Хувьцаа болон өмчийн бусад үнэт цаас гаргаснаас хүлээн авсан",accounts,
+account_cashflow_tag_313,"CF:[3.1.3] Төрөл бүрийн хандив",accounts,
+account_cashflow_tag_321,"CF:[3.2.1] Зээл, өрийн үнэт цаасны төлбөрт төлсөн мөнгө",accounts,
+account_cashflow_tag_322,"CF:[3.2.2] Санхүүгийн түрээсийн өглөгт төлсөн",accounts,
+account_cashflow_tag_323,"CF:[3.2.3] Хувьцаа буцаан худалдаж авахад төлсөн",accounts,
+account_cashflow_tag_324,"CF:[3.2.4] Төлсөн ногдол ашиг",accounts,
+account_cashflow_tag_325,"CF:[3.2.5] Төрөл бүрийн хандив, тусламж болон торгууль төлсөн",accounts,
+account_cashflow_tag_400,"CF:[4] Валютын ханшийн зөрүү",accounts,
+vat_report_tag2,ТТ3a: НӨАТ-аас чөлөөлөгдөх борлуулалтын орлого (2),taxes,base.mn
```

Comment:
Same as for account tags: we need to be quite careful about adding/removing tax tags in stable as this may break the databases of current customers.

This is because existing account.move.lines will still have the old tax tags on them, but the new VAT report will expect the new tax tags. So the VAT report will be incorrect unless the user manually changes the old tax tags into the new tax tags on their existing AMLs.